gradle/libs.versions.toml

```diff
@@ -100,3 +102,6 @@ jdgraphtCore = "org.jgrapht:jgrapht-core:1.5.1"
 
 detektPlugin = { module = "io.gitlab.arturbosch.detekt:detekt-gradle-plugin", version.ref = "detekt" }
 detektFormatting = { module = "io.gitlab.arturbosch.detekt:detekt-formatting", version.ref = "detekt" }
+
+[bundles]
+hamcrest = ["hamcrest", "hamcrestLibrary", "hamcrestCore"]
```

Comment:
`hamcrest-library` depends on `hamcrest-core`: https://repo1.maven.org/maven2/org/hamcrest/hamcrest-library/2.2/hamcrest-library-2.2.pom
Why do we add transitive dependency explicitly to clients?

Reply:
There is no reason to do that, thanks